java-util/src/main/java/io/druid/java/util/common/StringUtils.java

```diff
@@ -100,7 +100,20 @@ public static String fromUtf8(final ByteBuffer buffer)
     }
   }
 
-  public static String safeFormat(String message, Object... formatArgs)
+  /**
+   * Equivalent of String.format(Locale.ENGLISH, message, formatArgs).
+   */
+  public static String format(String message, Object... formatArgs)
+  {
+    return String.format(Locale.ENGLISH, message, formatArgs);
+  }
+
+  /**
+   * Formats the string as {@link #format(String, Object...)}, but instead of failing on illegal format, returns the
+   * concatenated format string and format arguments. Should be used for unimportant formatting like logging,
+   * exception messages, typically not directly.
+   */
+  public static String unimportantSafeFormat(String message, Object... formatArgs)
```

Comment:
My 2¢: the "safe" was originally meant to convey the behavior, so if it fails at this, then it doesn't need to stay. Consider something like `nonStrictFormat` or `lenientFormat` (since "non-strict" and "lenient" probably convey the behavior better than "safe").